com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/ShaderPass.template

```diff
@@ -24,6 +24,12 @@ Pass
     $splice(PassKeywords)
     $splice(GraphKeywords)
 
+    // Dots Instancing Defines
+    $splice(DotsInstancingOptions)
+
+    // Graph Instanced properties (must be included before UnityInstancing.hlsl)
+    $splice(HybridV1InjectedBuiltinProperties)
+
```

Comment:
These macros must be defined before ShaderVariables.hlsl is included (which includes UnityInstancing.hlsl)

Reply:
I hope it will not introduce other issues. Note that hybrid test are disabled on master (for various reasons), so it need to be manually checked

Reply:
I expect the regression broke Hybrid as well. . but I can test it manually.

Reply:
The macro definitions here only rely on the INSTANCING_ON keyword and the platform defines -- so it should be fairly save to move them earlier.

Reply:
I was thinking we should add some simple instanced tests to HDRP, to ensure this continues to work in the future.

Reply:
yes good idea

Reply:
FYI - hybrid was not broken by the regression, as they actually use a different way to declare the instanced properties.

I had to fix up the hybrid property access macros to be compatible.